src/utils/types.ts

```diff
@@ -28,3 +28,15 @@ export const arrayIncludes = <T>(arr?: T[], element?: T | null): boolean => {
 };
 
 export const urlRegex = /((([A-Za-z]{3,9}:(?:\/\/)?)(?:[-;:&=+$,\w]+@)?[A-Za-z0-9.-]+|(?:www\.|[-;:&=+$,\w]+@)[A-Za-z0-9.-]+)((?:\/[+~%/.\w\-_]*)?\??[-+=&;%@.\w_]*#?[.!/\\\w]*)?)/;
+
```

Comment:
Maybe move all of this in `./src/types/branding.ts`.

Reply:
If there was more to this then I would agree. I'm not convinced by having a new module for a single line of actual code.